libsolidity/ast/Types.cpp

```diff
@@ -561,6 +812,75 @@ shared_ptr<IntegerType const> IntegerConstantType::integerType() const
 		);
 }
 
+shared_ptr<FixedPointType const> RationalNumberType::fixedPointType() const
+{
+	//do calculations up here
```

Comment:
Why would we solAssert that integerbits be bigger than fractionalBits? That takes away the ability to have a fixed0x256.